examples/todo/tests/test_suite.py

```diff
@@ -8,7 +8,16 @@
 from . import ToDoBox
 
 
-class ToDoTest(BBTestCase):
+class BaseToDoTest(BBTestCase):
```

Comment:
This is an example. Our simplest example and this class complicates things. I prefer to keep this kind of tricks for a more complex example.